### PR TITLE
fix: fix incorrect log message in info_collector::on_app_stat

### PR DESCRIPTION
**Pull request by Smityz**
fix incorrect log message in info_collector::on_app_stat

### CODE REVIEW

**Comment by acelyc111** [comment]
> In the past version, stat table is considered a table, but in some cases, it will cause some misunderstanding. If there we drop the table stat and temp(which are initially generated by default), app_count will be -1.

The description let me puzzled, I think you don't need to explain so much :)

File: src/server/info_collector.cpp [hunk 1]
```diff
@@ -163,7 +163,7 @@ void info_collector::on_app_stat()
     get_app_counters(all_stats.app_name)->set(all_stats);
 
     ddebug("stat apps succeed, app_count = %d, total_read_qps = %.2f, total_write_qps = %.2f",
```
Comment (levy5307) [inline src/server/info_collector.cpp:165]: `ddebug_f`

**Comment by levy5307** [comment]
> > In the past version, stat table is considered a table, but in some cases, it will cause some misunderstanding. If there we drop the table stat and temp(which are initially generated by default), app_count will be -1.
> 
> The description let me puzzled, I think you don't need to explain so much :)

I agree :(

File: src/server/info_collector.cpp [hunk 2]
```diff
@@ -162,10 +162,10 @@ void info_collector::on_app_stat()
     }
     get_app_counters(all_stats.app_name)->set(all_stats);
 
-    ddebug("stat apps succeed, app_count = %d, total_read_qps = %.2f, total_write_qps = %.2f",
-           (int)(all_rows.size() - 1),
-           all_stats.get_total_read_qps(),
-           all_stats.get_total_write_qps());
+    ddebug_f("stat apps succeed, app_count = {}, total_read_qps = {}, total_write_qps = {}",
+             (int)(all_rows.size()),
```
Comment (neverchanje) [inline src/server/info_collector.cpp:166]:
```suggestion
             all_rows.size(),
```

**Comment by hycdong** [comment]
I have another question, why original log use `all_rows.size() -1` , it is to exclude test table or stat table?

**Comment by Smityz** [comment]
> I have another question, why original log use `all_rows.size() -1` , it is to exclude test table or stat table?

Through the comparison of historical versions, I speculate that it was a mistake made during reconstruction/
https://github.com/XiaoMi/pegasus/blob/v1.11/src/server/info_collector.cpp#L140

**Pull request merged**